src/presentation/pages/Note.vue

```diff
@@ -79,11 +83,37 @@ const editor = ref<typeof Editor | undefined>(undefined);
  *
  * @param data - editor data
  */
-function noteChanged(data: NoteContent): void {
+async function noteChanged(data: NoteContent): Promise<void> {
   const isEmpty = editor.value?.isEmpty();
 
+  let updatedNoteCover: Blob | null = null;
+
+  /**
+   * Get html element with note
+   */
+  const editorElement = document.getElementById('editorjs');
+
+  /**
+   * If it does not exist, we can not do screenshot
+   */
+  if (editorElement !== null) {
+    updatedNoteCover = await makeElementScreenshot(editorElement, {
```

Comment:
its better to move it inside the `!isEmpty()` statement

Reply:
done